server/commands/write_authzmodel.go

```diff
@@ -30,40 +30,37 @@ func NewWriteAuthorizationModelCommand(
 
 // Execute the command using the supplied request.
 func (w *WriteAuthorizationModelCommand) Execute(ctx context.Context, req *openfgapb.WriteAuthorizationModelRequest) (*openfgapb.WriteAuthorizationModelResponse, error) {
-	typeDefinitions := req.GetTypeDefinitions()
-
 	// Until this is solved: https://github.com/envoyproxy/protoc-gen-validate/issues/74
-	if len(typeDefinitions) > w.backend.MaxTypesInTypeDefinition() {
+	if len(req.GetTypeDefinitions()) > w.backend.MaxTypesInTypeDefinition() {
 		return nil, serverErrors.ExceededEntityLimit("type definitions in an authorization model", w.backend.MaxTypesInTypeDefinition())
 	}
 
-	schemaVersion, err := typesystem.NewSchemaVersion(req.GetSchemaVersion())
-	if err != nil {
+	switch req.SchemaVersion {
+	case "", "1.0":
```

Comment:
Can we put this validation logic as part of the logic of `typesystem.Validate(model)` instead? 

Makes our separation of concerns more well organized.

Reply:
Done. Thanks!